Tech-Journal-ConsoleApp/Tech-Journal-ConsoleApp/SendEmail.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+
+namespace Tech_Journal_ConsoleApp
+{
+    class SendEmail
+    {
+        public const string Path = "c:\\temp\\appsettings.txt";
+
+        private bool _validEmail;
+        private string _toThisEmailAddress;
+        private string _emailUsername;
+        private string _emailPassword;
+
+
+        public SendEmail()
+        {
+            _validEmail = false;
+        }
+        public void ValidateEmailSettings()
+        {
+            if (!File.Exists(Path))
+            {
+                using (var sw = File.AppendText(Path))
+                {
+                    Console.WriteLine("\nEmail Settings were not found. Creating file to store email settings.");
```

Comment:
Question: Do you need to add `\n` on every Console.Writeline? Because it's writing a line, you should ideally not need to specify a line break. Right?